installing/installing_gcp/installing-gcp-user-infra-vpc.adoc

```diff
@@ -21,16 +21,10 @@ The steps for performing a user-provisioned infrastructure installation are prov
 
 == Prerequisites
 
-* Review details about the
-xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
-processes.
-* If you use a firewall and plan to use telemetry, you must
-xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure the firewall to allow the sites] that your cluster requires access to.
-* If you do not allow the system to manage identity and access management (IAM),
-then a cluster administrator can
-xref:../../installing/installing_gcp/manually-creating-iam-gcp.adoc#manually-creating-iam-gcp[manually
-create and maintain IAM credentials]. Manual mode can also be used in
-environments where the cloud IAM APIs are not reachable.
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* If you use a firewall and plan to use telemetry, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured the firewall to allow the sites] that your cluster requires access to.
```

Comment:
No prereq to configured GCP account here?

Reply:
Following your comment, I reviewed the GCP installation assemblies and am not certain about whether the prerequisite has been intentionally left out. I would like to suggest that this is reviewed outside of the scope of this pull request, which mainly focuses on updating the parallelism for existing prerequisites.

One thing I have subsequently done though is change any existing "You configured a GCP account to host the cluster" prerequisites to "You configured a GCP project to host the cluster", which corresponds to the assembly that the prerequisites link to.

@EricPonvelle, externally to this PR, can you please review whether the "You configured a GCP project to host the cluster" prerequisite should be added to the GCP installation assemblies that do not include it? Thanks!

Reply:
@gpei Could you provide guidance if these prerequisites should be included in all of the assemblies?

Reply:
Is it possible we could have a preview on the changes, to be honest, I don't quite understand what this change is about, not familiar with the doc modules usage...

Reply:
@gpei this PR, which is now merged, updates the bullet points in the top level "Prerequisites" sections across the installation book, so that they all have the same format / tense. It also adds a prerequisite link to the "Selecting a cluster installation method and preparing it for users" page.

The question in this specific comment is about whether the the "You configured a GCP project to host the cluster" prerequisite that is in some of the OCP on GCP installation assemblies (for example, [this one](https://docs.openshift.com/container-platform/4.8/installing/installing_gcp/installing-gcp-default.html#prerequisites)), should be in the "Prerequisites" section in all of the OCP on GCP installation assemblies. I.e. is a GCP project a base requirement for all OCP on GCP installations? If so, is that prerequisite handled elsewhere in the installation assemblies that do not include it as a top level prerequisite? Thanks!

Reply:
@gpei thank you for confirming these points!